client/src/main/java/io/getstream/chat/android/client/notifications/FirebaseMessageParserImpl.kt

```diff
@@ -24,15 +23,13 @@ internal class FirebaseMessageParserImpl(val handler: ChatNotificationHandler) :
     }
 
     private fun verifyPayload(message: RemoteMessage): Boolean {
-
         val messageId = message.data[messageIdKey]
         val channelId = message.data[channelIdKey]
         val channelType = message.data[channelTypeKey]
 
-        return message.data.containsKeys(messageIdKey, channelTypeKey, channelIdKey) && !isNullOrEmpty(
-            messageId,
-            channelId,
-            channelType
-        )
+        return message.data.containsKeys(messageIdKey, channelTypeKey, channelIdKey) &&
+            !messageId.isNullOrEmpty() &&
+            !channelId.isNullOrEmpty() &&
+            !channelType.isNullOrEmpty()
```

Comment:
We could get rid of all of our custom extensions with something like

```kotlin
val keys = setOf(messageIdKey, channelIdKey, channelTypeKey)
return message.data.keys.containsAll(keys) && keys.none { key -> message.data[key].isNullOrEmpty() }
```

I think with this we could remove `LangExtensions.kt` and its tests, too.